tools/linter/adapters/pip_init.py

```diff
@@ -52,7 +52,10 @@ def run_command(args: list[str]) -> subprocess.CompletedProcess[bytes]:
         stream=sys.stderr,
     )
 
-    uv_available = shutil.which("uv") is not None
+    uv_available = (
+        any(prefix in sys.base_prefix for prefix in ["uv/python", "uv\\python"])
```

Comment:
lol is uv\\python for windows

Reply:
Yes, I guess so, but have no way of testing whether it'll work or not